judo-ui-react/src/main/resources/actor/src/containers/components/link.tsx.hbs

```diff
@@ -39,16 +40,75 @@ export function {{ componentName link }}(props: {{ componentName link }}Props) {
   const { ownerData, actions, storeDiff, submit, validationError, disabled, readOnly, editMode, isLoading, isDraft } = props;
   const { t } = useTranslation();
 
+
+  const aggregationInputButtonProps: AggregationInputButtonProps[] = [
```

Comment:
Please wrap this in `useMemo`, also pay attention to the dependencies array!

Reply:
[ec0c04e](https://github.com/BlackBeltTechnology/judo-ui-react-template/pull/378/commits/ec0c04ef0c9e729799ce08472123e586dede906b)